codegen/templates/http_client.tmpl

```diff
@@ -167,7 +176,11 @@ func (c *{{$clientName}}) {{$methodName}}(
 	}
 	{{- end}}
 
-	res, err := req.Do()
+	var res *zanzibar.ClientHTTPResponse
+	err = hystrix.Do("{{$clientID}}", func() error {
```

Comment:
We should call `hystrix.DoC` with the proper context here, so the context is respected.